content/docs/1.10.0/advanced-resources/rebuilding/offline-replica-rebuilding.md

```diff
@@ -29,6 +29,21 @@ Starting with v1.9.0, Longhorn supports offline replica rebuilding, allowing deg
 - When triggered, Longhorn attaches the volume without activating the frontend, rebuilds any missing replicas, and then detaches the volume upon completion.
 - This process can be interrupted if the workload scales up.
 
+## Resource Constraints
+
+If these requirements are not met, Longhorn will consider the cluster to have insufficient resources:
+
+- All instance managers on schedulable nodes must be ready.
+- The scheduler verifies that there is an available disk with sufficient space on a schedulable node for rebuilding a replica.
```

Comment:
Again, the doc is for readers that don't know the context of the feature. Thus, it is better to organize the sentences as the steps

- Why is this section needed?
  - Problem
  - Proposed solution
  - Expected benefit
- How does it work?
  - High-level workflow
  - Key components involved
  - Example scenario
- Limitations and considerations
  - Known issues / trade-offs
  - Compatibility notes

You don't need to go into deep technical details, but you should clearly convey the feature and its importance to the users.

Reply:
cc @sushant-suse